### ISSUE TITLE
Steam overlay began to automatically close linux

### ISSUE THREAD

**Issue by Yanpas** [ValveSoftware/steam-for-linux#3981]
When I do something in overlay browser, especially scrolling, cs go closes steam overlay automatically (or overlay closes itself) 3-4 weeks ago there were no such problem.
https://gist.github.com/Yanpas/afa5682d8a81876a66fc

**Comment by directhex** [comment]
Yeah, the desktop overlay is a total disaster at the moment. I've given up, I can't use it at all.

**Comment by UnknownFearNG** [comment]
Having this same issue as well.
